gammapy/modeling/models/temporal.py

```diff
@@ -934,14 +935,14 @@ class TemplatePhaseCurveTemporalModel(TemporalModel):
     A timing solution is used to compute the phase corresponding to time and
     a template phase curve is used to determine the associated ``norm``.
 
-    The phasecurve is given as a table with columns ``phase`` and ``norm``.
+    The phase curve is given as a table with columns ``phase`` and ``norm``.
 
     The ``norm`` is supposed to be a unit-less multiplicative factor in the model,
     to be multiplied with a spectral model.
 
     The model does linear interpolation for times between the given ``(phase, norm)`` values.
 
-    The implementation currently uses `scipy.interpolate. InterpolatedUnivariateSpline`,
+    The implementation currently uses `scipy.interpolate.InterpolatedUnivariateSpline`,
```

Comment:
```suggestion
    The implementation currently uses `~scipy.interpolate.InterpolatedUnivariateSpline`,
```

Reply:
It was like this, but @Astro-Kirsty prefered like this...

Reply:
OK. The change I propose would remove the full path in the doc, while here I think we need it. So don't apply.